src/components/TextField/TextField.driver.ts

```diff
@@ -62,3 +69,18 @@ export const textFieldDriverFactory = ({ element, eventTrigger }) => {
     },
   };
 };
+
+export interface TextFieldDriver extends BaseUniDriver {
+  isSuccessIconExist(): Promise<boolean>;
+  isSuccess(): Promise<boolean>;
+}
+
+export const textFieldUniDriverFactory = (base: UniDriver): TextFieldDriver => {
+  const stylableUtil = new StylableUnidriverUtil(style);
```

Comment:
We actually prefer to not use StylableUnidriverUtil, it causes some problems in e2e tests.
Also, of course we prefer using Unidrivers, but in this case, since we're using a driver from wix-ui-core, and also we have automatic utilities exporting these drivers.

So it's best to move these methods to the driver above, instead of creating a new driver.